### PR TITLE
be community build friendly

### PR DESCRIPTION
**Pull request by SethTisue**
the main change is 1289659 which fixes scala/scala-dev#381

but while I was here, I girl-scouted some other stuff

### CODE REVIEW

**Comment by farmdawgnation** [comment]
Welp, I removed the methods from lift-json that broke everything so I suppose it's only appropriate that I review this! Hah!

Thanks for the contribution. I'll take a look ASAP so the community build gets healthy again.

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,3 +1,5 @@
+scalacOptions += "-Xfatal-warnings"
+
```
Comment (farmdawgnation) [inline project/plugins.sbt]: Why did this end up in the plugins file?

Comment (SethTisue) [inline project/plugins.sbt]: because after my changes, the build definition itself now compiles without producing any warnings

Comment (SethTisue) [inline project/plugins.sbt]: `plugins.sbt` is admittedly a bit misleading now as a filename given that there's now something else in there too, but it's kinda traditional

Comment (farmdawgnation) [inline project/plugins.sbt]: Ah, I see. The commit this originated in was porting to the new build.sbt format. Didn't quite put two and two together here.

**Pull request merged**